live-examples/wat-examples/numeric/extend.wat

```diff
@@ -0,0 +1,13 @@
+(module
+  (import "console" "log" (func $log (param i64)))
+  (func $main
+
+    i32.const 10 ;; push an i32 onto the stack
+
+    i64.extend_i32_u ;; extend from i32 to i64
```

Comment:
I suggest saying "zero-extend from i32 to i64", to emphasize how it's different from the `_s` form.

Reply:
Accepted your suggestion but changed to signed as I suspect that more people will understand `sign-extend` than `zero-extend`